tm/src/main/java/io/seata/tm/api/DefaultFailureHandlerImpl.java

```diff
@@ -67,6 +68,14 @@ public void onRollbackFailure(GlobalTransaction tx, Throwable cause) {
         timer.newTimeout(new CheckTimerTask(tx, GlobalStatus.Rollbacked), SCHEDULE_INTERVAL_SECONDS, TimeUnit.SECONDS);
     }
 
+    @Override
+    public void onRollbackRetrying(GlobalTransaction tx, Throwable cause) {
+        StackTraceLogger.info(LOGGER, cause, "Retrying to rollback transaction[{}]", new String[] {tx.getXid()}, null,
+            null);
```

Comment:
StackTraceLogger.info(LOGGER, cause, "Retrying to rollback transaction[{}]", new String[] {tx.getXid()}, "Retrying to rollback transaction[{}]", new String[] {tx.getXid()});

Reply:
> StackTraceLogger.info(LOGGER, cause, "Retrying to rollback transaction[{}]", new String[] {tx.getXid()}, "Retrying to rollback transaction[{}]", new String[] {tx.getXid()});

PTAL @zjinlei